docs/learn.html

```diff
@@ -162,9 +162,10 @@ <h5 class="my-2">
             </div>
             <p>
               If you are a sysadmin or DevOps engineer, learn how to
-              <a href="remote-deploying-codewind.html"
-                >deploy Codewind remotely</a
-              >.
+              <a href="remote-deploying-codewind.html">deploy Codewind remotely</a>, or learn how to
+              <a href="che-installinfo.html">
+                install Codewind for Eclipse Che.
+              </a>
```

Comment:
`...Che.`  Suggest we move the full stop outside of the link, so becomes `</a>.`